packages/lib/test/defaultStyles.spec.js

```diff
@@ -0,0 +1,107 @@
+import { expect } from 'chai';
+import _ from 'lodash';
```

Comment:
Do not import the whole lodash package (also remove from package.json).
Implement the needed functions here: 
https://github.com/wix/pro-gallery/blob/master/packages/lib/src/common/utils/lodash.js
and import it.

Reply:
It's a dev dependency therefor doesn't effect bundle size.